pkg/plugins/scms/github/pullrequest.go

```diff
@@ -83,6 +91,27 @@ type PullRequest struct {
 	remotePullRequest PullRequestApi
 }
 
+// isMergeMethodValid ensure that we specified a valid merge method.
+func isMergeMethodValid(method string) (bool, error) {
+	if len(method) == 0 ||
```

Comment:
```suggestion
	if method == "" ||
```

NITPICK: "Idiomatic" way to check for empty strings: when reading it clear that `method` is a string (while `len(method)` is ambiguous: is it a slice or a string or something else ?)